.vscode/settings.json

```diff
@@ -1,8 +1,11 @@
 {
```

Comment:
Can you revert the changes to this before merge?

Reply:
I will do this at the end